llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1583,6 +1585,50 @@ class VPWidenCallRecipe : public VPSingleDefRecipe {
 #endif
 };
 
+/// A recipe representing a sequence of load -> update -> store as part of
+/// a histogram operation. This means there may be aliasing between vector
+/// lanes, which is handled by the llvm.experimental.vector.histogram family
+/// of intrinsics. The only update operations currently supported are
+/// 'add' and 'sub' where the other term is loop-invariant.
+class VPHistogramRecipe : public VPRecipeBase {
+  unsigned Opcode;
```

Comment:
nit: document that this is the opcode of the increment?

Reply:
done